lms/djangoapps/branding/tests/test_page.py

```diff
@@ -214,22 +214,22 @@ def test_course_cards_sorted_by_default_sorting(self):
         self.assertEqual(template, 'courseware/courses.html')
 
         # Now the courses will be stored in their announcement dates.
```

Comment:
I *think* comment should say "sorted" not "stored". Also, I think that we should update the comment to reflect why the order has changed